common/src/main/java/com/wynntils/wynn/model/CharacterManager.java

```diff
@@ -125,10 +147,14 @@ public static final class CharacterInfo {
         // This was implemented the same way by legacy.
         private final int id;
 
-        private CharacterInfo(ClassType classType, boolean reskinned, int level, int id) {
+        private final ProfessionInfo professionInfo;
+
+        private CharacterInfo(
+                ClassType classType, boolean reskinned, int level, int id, ProfessionInfo professionInfo) {
```

Comment:
This makes me realize that we do not update our CharacterInfo, iirc. So any level up, either in combat level or professions, will not be recorded, and thus the quest book requirements will be wrong. This needs to be fixed, but perhaps not in this PR.

Reply:
https://github.com/Wynntils/Artemis/issues/416

Reply:
Thanks for opening a bug. I was thinking it was needed but at the moment it was just too much work for me :)